test/db/index.js

```diff
@@ -0,0 +1,449 @@
+        event = await DBHelper.findEvent({address: createdEvent.address});
+        event.length.should.equal(1);
+        expect(event[0]).excluding(["_id", 'status']).to.deep.equal(createdEvent);
+        event[0].status.should.equal(EVENT_STATUS.ORACLE_RESULT_SETTING)
+      })
+
+      it('should update event status to OPEN_RESULT_SETTING', async () => {
+        await DBHelper.updateEventStatusOpenResultSetting(createdEvent.resultSetEndTime);
+        event = await DBHelper.findEvent({address: createdEvent.address});
+        event.length.should.equal(1);
+        expect(event[0]).excluding(["_id", 'status']).to.deep.equal(createdEvent);
+        event[0].status.should.equal(EVENT_STATUS.OPEN_RESULT_SETTING)
+      })
+
+      it('should update event status to ARBITRATION', async () => {
+        createdEvent = {"txType":"CREATE_EVENT","txid":"0xf0e68c3cfa4e77f0a5b46afd9f8c5efccd5d90f419053c6d452b021bc203c44f","txStatus":"SUCCESS","blockNum":3979339,"address":"0xfef4a675dba91b91608dc75c40deaca0333af514","ownerAddress":"0x47ba776b3ed5d514d3e206ffee72fa483baffa7e","version":5,"name":"QQQ","results":["Invalid","2","1"],"numOfResults":3,"centralizedOracle":"0x47ba776b3ed5d514d3e206ffee72fa483baffa7e","betStartTime":1560292822,"betEndTime":1560379222,"resultSetStartTime":1560379222,"resultSetEndTime":1560465622,"escrowAmount":"100000000","arbitrationLength":300,"thresholdPercentIncrease":"10","arbitrationRewardPercentage":10,"currentRound":1,"currentResultIndex":2,"consensusThreshold":"1100000000","arbitrationEndTime":1561586376,"status":"OPEN_RESULT_SETTING","language":"en-US"}
+        await DBHelper.insertEvent(createdEvent);
+        await DBHelper.updateEventStatusArbitration(createdEvent.arbitrationEndTime - 1);
+        event = await DBHelper.findEvent({address: createdEvent.address});
+        event.length.should.equal(1);
+        expect(event[0]).excluding(["_id", 'status']).to.deep.equal(createdEvent);
+        event[0].status.should.equal(EVENT_STATUS.ARBITRATION)
+      })
+
+      it('should update event status to WITHDRAWING', async () => {
+        await DBHelper.updateEventStatusWithdrawing(createdEvent.arbitrationEndTime);
+        event = await DBHelper.findEvent({address: createdEvent.address});
+        event.length.should.equal(1);
+        expect(event[0]).excluding(["_id", 'status']).to.deep.equal(createdEvent);
+        event[0].status.should.equal(EVENT_STATUS.WITHDRAWING)
+      })
+
+      describe('test event withdrawn list', () => {
+        const withdraw = {
+          eventAddress: "0xfef4a675dba91b91608dc75c40deaca0333af514",
+          winnerAddress: "0x47ba776b3ed5d514d3e206ffee72fa483baffa7e",
+        }
+        it('should update event withdrawnlist', async () => {
+          await DBHelper.updateEventWithdrawnList(withdraw);
+          event = await DBHelper.findOneEvent({address: withdraw.eventAddress});
+          event.withdrawnList[0].should.equal(withdraw.winnerAddress);
+        })
```

Comment:
missing updateEventWithdrawnList tests:
1. does not add to withdrawnList if already on the list